app/controllers/albums.js

```diff
@@ -1,13 +1,46 @@
 const logger = require('../logger'),
-  bcrypt = require('bcryptjs'),
   albumService = require('../services/albums'),
-  User = require('../models').users,
-  userMw = require('../middlewares/users'),
   errors = require('../errors'),
-  fetch = require('node-fetch');
+  Album = require('../models').albums_bought;
 
 exports.getAll = (request, response, next) => {
   if (!request.userLogged)
     return response.status(400).send(errors.authenticationError(['You must be logged to see all albums']));
   albumService.getAll(response).then(data => response.status(200).send(data));
 };
+
+exports.buy = (request, response) => {
+  if (!request.userLogged)
+    return response.status(400).send(errors.authenticationError('You must be logged to buy albums'));
+  const handleError = (err, message) => {
```

Comment:
You should define this outside of this the `buy()` mtehod, in case you need it somewhere else